contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_mac.py

```diff
@@ -0,0 +1,131 @@
+import re
+from typing import Optional
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.execution_engine import PandasExecutionEngine
+from great_expectations.expectations.expectation import ColumnMapExpectation
+from great_expectations.expectations.metrics import (
+    ColumnMapMetricProvider,
+    column_condition_partial,
+)
+
+MAC_REGEX = r"^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})|([0-9a-fA-F]{4}\\.[0-9a-fA-F]{4}\\.[0-9a-fA-F]{4})$"
+
+
+# This class defines a Metric to support your Expectation.
+# For most ColumnMapExpectations, the main business logic for calculation will live in this class.
+class ColumnValuesToBeValidHexColor(ColumnMapMetricProvider):

```

Comment:
`ColumnValuesToBeValidMac`?

Reply:
Exactly! I was in a mess when working on multiple branches. Will be more careful next time!